Default configuration of Kibana dashboards and indexes (#126) (#2424)
epiphany-platform/epiphany: docs/home/howto/LOGGING.md

  cloud:
    k8s_as_cloud_service: true
```

## How to use default Kibana dashboards

> Default Filebeat configuration sets this option to 'auto'. In this case if Kibana service exists it will be enabled - false otherwise.
> Other possible values are True and False.

Reviewer: ```suggestion
It is possible to configure `setup.dashboards.enabled` and `setup.dashboards.index` Filebeat settings using `specification.kibana.dashboards` key in `configuration/filebeat` doc.

When `specification.kibana.dashboards.enabled` is set to `auto`, the corresponding setting in Filebeat configuration file will be set to `true` only if Kibana is configured to be present on the host.
Other possible values are `true` and `false`.
```

Default configuration:
```
specification:
  kibana:
    dashboards:

>       enabled: 'auto'
>       index: 'filebeat-*'

Reviewer: ```suggestion
      enabled: auto
      index: filebeat-*
```

```

> Setting this option to True and not providing Kibana will result in a Filebeat crash.

Reviewer: ```suggestion
Note: Setting `specification.kibana.dashboards.enabled` to `true` not providing Kibana will result in a Filebeat crash.
```
